synapse/http/server.py

```diff
@@ -327,10 +327,6 @@ async def _async_render_wrapper(self, request: "SynapseRequest") -> None:
             # of our stack, and thus gives us a sensible stack
             # trace.
             f = failure.Failure()
-            logger.exception(
-                "Error handling request",
-                exc_info=(f.type, f.value, f.getTracebackObject()),
-            )
```

Comment:
If we remove this, will we still have visibility of errors (e.g. 500s) in the application that need investigation?

Reply:
Yes, we log them elsewhere.

To be clear: This entire block was added in #15913, so we're essentially just reverting that change

Reply:
I think it was a bit of debugging I added that crept into the final PR